cypress/integration/persianBodySpec.js

```diff
@@ -15,21 +15,12 @@ describe('Article Body Tests', () => {
     cy.visit('/persian/articles/cwv2xv848j5o');
   });
 
-  it('should render a headline', () => {
-    checkElementStyles(
-      'h1',
-      'پهپادی که برایتان قهوه می‌آورد',
-      'rgb(34, 34, 34)',
-      'ReithSerifNewsMedium, Helvetica, Arial, sans-serif',
-    );
+  it('should render an H1, which contains/displays a styled headline', () => {
+    headlineDataWindow('headline');
```

Comment:
The arguments for these are always the same so probably safe to just hardcode it in the fuction